packages/owl-bot/src/core.ts

```diff
@@ -126,7 +126,8 @@ export async function triggerPostProcessBuild(
     );
     return null;
   }
-
+  if (!prData?.head?.repo?.full_name)
```

Comment:
similarly, there was a breaking change in a patch for octokit, which this addresses.